caas/kubernetes/provider/bootstrap.go

```diff
@@ -60,6 +64,12 @@ var (
 const (
 	mongoDBContainerName   = "mongodb"
 	apiServerContainerName = "api-server"
+
```

Comment:
I think we can do better here. The goal of probes is to indicate to the underlying scheduling system when something is ready and not ready to receive data and also restart a failed program.

I would suggest the following values:
```
apiServerProbeInitialDelay = 1
apiServerProbeTimeout = 3
apiServerProbePeriod = 5
apiServerProbeSuccess = 1
apiServerProbeFailure = 2
```

If you want to focus on making bootstrap quicker then I would look at creating a startupProbe config for the container that can hit pebble quickly to check it's running.

Liveness and readiness probes are not the place to fix bootstrap issues they're for maintaining stable production systems.

Reply:
Sounds good. I can copy the current values to a startup probe, and use your suggested values for liveness/readiness?